src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -155,6 +167,11 @@ const AugmentedRealityView = ({ sceneNavigator }) => {
         onLoadStart={handleLoadStart}
         onLoadEnd={handleLoadEnd}
         onError={(event) => handleError(event)}
+        animation={{
+          loop: true,
+          name: object.animationName,
+          run: startAnimation
+        }}
```

Comment:
could there be a problem if we have no `animationName` here?

Reply:
If there is no animation name, the 3D object just stays as it is loaded and no error occurs.